conda/auxlib/type_coercion.py

```diff
@@ -263,3 +265,23 @@ def typify_data_structure(value, type_hint=None):
 
 def maybecall(value):
     return value() if callable(value) else value
+
+
+def is_only_subclass(obj_class: Union[Any, tuple[Any]], base_class: type) -> bool:
```

Comment:
I think "strict" is a better term here.

```suggestion
def is_strict_subclass(obj_class: Union[Any, tuple[Any]], base_class: type) -> bool:
```